src/wtforms/meta.py

```diff
@@ -37,13 +37,21 @@ def wrap_formdata(self, form, formdata):
         :param formdata: Form data.
         :return: A form-input wrapper compatible with WTForms.
         """
+
         if formdata is not None and not hasattr(formdata, "getlist"):
             if hasattr(formdata, "getall"):
+                # Presence of getall attribute implies Webob-style multidict
+                if len(formdata) == 0:
```

Comment:
As @hyperknot pointed out in https://github.com/wtforms/wtforms/issues/460#issuecomment-489437421, checking for zero-length wouldn't necessarily be safe as this could break for valid but empty form submissions.